ui/apps/ui/src/app/components/results-with-pagination/result.component.ts

```diff
@@ -50,7 +51,9 @@ import { RedirectService } from '@collections/services/redirect.service';
           ><img src="/assets/usage-date.svg" />
           <ng-container i18n>{{ date }}</ng-container></span
         >
-        <span *ngIf="type !== null" class="statistic text-muted"
+        <span
+          *ngIf="type !== null && type.label !== 'interoperability guideline'"
```

Comment:
Is it the only way to determine guideline? maybe using `value` would be better?

Reply:
I used value.